### PR TITLE
Add test for display of credits page

### PR DESCRIPTION
**Pull request by labkey-sweta**
#### Rationale
Test coverage for Add test for display of credits page
https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=46587

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

File: src/org/labkey/test/pages/core/admin/ShowAdminPage.java [hunk 1]
```diff
@@ -230,6 +237,7 @@ protected class ElementCache extends LabKeyPage.ElementCache
         protected WebElement systemMaintenanceLink = Locator.linkWithText("system maintenance").findWhenNeeded(this);
         protected WebElement systemPropertiesLink = Locator.linkContainingText("system properties").findWhenNeeded(this);
         protected WebElement viewsAndScriptingLink = Locator.linkWithText("views and scripting").findWhenNeeded(this);
+        protected WebElement creditsLink = Locator.linkWithText("credits").findElementOrNull(this);
```
Comment (labkey-tchad) [inline src/org/labkey/test/pages/core/admin/ShowAdminPage.java]: I'm guessing auto-complete picked the wrong method here.
```suggestion
        protected WebElement creditsLink = Locator.linkWithText("credits").findWhenNeeded(this);
```

File: src/org/labkey/test/tests/AdminConsoleCreditsTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        Test coverage : Issue 46587: Add test for display of credits page
+        https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=46587
+     */
+    @Test
+    public void testAdminConsoleCredits()
+    {
+        goToAdminConsole().clickCredits();
+        log("Verifying the page is properly loaded");
+        assertTextPresent("JAR Files Distributed with the API Module");
+    }
```
Comment (labkey-tchad) [inline src/org/labkey/test/tests/AdminConsoleCreditsTest.java]: Add this to the existing `AdminConsoleTest` instead of creating a whole new test class.

**Review by labkey-tchad — APPROVED** [review]
Just a couple of notes

File: src/org/labkey/test/tests/AdminConsoleCreditsTest.java (hunk 2 above)
Comment (labkey-sweta) [reply src/org/labkey/test/tests/AdminConsoleCreditsTest.java]: done

File: src/org/labkey/test/pages/core/admin/ShowAdminPage.java (hunk 1 above)
Comment (labkey-sweta) [reply src/org/labkey/test/pages/core/admin/ShowAdminPage.java]: done

**Pull request closed without merging**